src/frontend/replayserver.cc

```diff
@@ -135,7 +136,16 @@ int main( void )
         }
 
         if ( best_score > 0 ) { /* give client the best match */
-            cout << HTTPResponse( best_match.response() ).str();
+            string best_response = HTTPResponse( best_match.response() ).str();
+            // Remove first line if status line is first
+            int pos = best_response.find( "\n" );
+            string first_line = best_response.substr( 0, pos );
+            if ( first_line.find( "HTTP" ) == 0 ) {
+                int start_pos = first_line.find( ' ' );
+                cout << "Status: " << first_line.substr( start_pos+1, 3 );
```

Comment:
Apache is parsing the other headers replayserver prints...it's just that it doesn't seem to play well with the status line as it's printed currently (`HTTP/1.1 200 OK`) and seems to want something in the format `Status: 200`.
I think it's possible that in the nph cgi handler, some other method call handles the parsing of that first status line...still reading through the source code though, will get back to  you as soon as I understand what's going on.